articles/tokens/id-token.md

```diff
@@ -0,0 +1,88 @@
+---
+title: ID token
+description: How to obtain, use and renew an id_token.
+---
+
+# ID Token
+
+## Overview
+
+The ID token, usually referred to as `id_token` in code samples, is a [JSON Web Token (JWT)](/jwt) that contains user profile attributes represented in the form of _claims_. These claims are statements about the user, which can be trusted if the consumer of the token can verify its signature (which was generated with the Auth0 client's _Client Secret_). The `id_token` is consumed by the client and used to get user information like the user's name, email, and so forth, typically used for UI display. It was added to the OIDC specification as an optimization so the client can know the identity of the user, without having to make an additional network requests.
+
+The `id_token` conforms to an industry standard (IETF [RFC 7519](https://tools.ietf.org/html/rfc7519)) and contains three parts: a header, a body and a signature.
+- The header contains the type of token and the hash algorithm used on the contents of the token.  
+- The body, also called the payload, contains identity claims about a user.  There are some claims with registered names, for things like the issuer of the token, the subject of the token (who the claims are about), and the time of issuance.  Any number of additional claims with other names can be added, though care must be taken to keep the JWT within the browser size limitations for URLs.  
```

Comment:
ID tokens are not always returned in URLs, for example when using `response_type=code` or `response_mode=form_post`. When using these modes it doesn't really matter how large the ID token becomes, because it won't be sent anywhere (the access token, if any, will be used to perform API requests).